client/src/components/new-request/analyze-results.vue

```diff
@@ -244,8 +247,36 @@ export default class AnalyzeResults extends Vue {
     }
   }
 
+  get allDesignationsStripped () {
+    return this.stripAllDesignations(this.originalName)
+  }
+  get allowProceed () {
+    /* newReqModule.designationIssueTypes is a list of all the designation-related issue types.  designationIsFixed
+     will not be true unless one of these issues was solved.  some sets of issues will not include a designation-related
+     issue and this will need to return true in these cases even tho designationIsFixed will be false */
+    if (!this.hasDesignationIssue) {
+      if (this.isLastIndex) {
+        return true
+      }
+    }
+    return this.designationIsFixed
+  }
+  get baseWordsAreUnchanged () {
+    let nameTest = this.stripAllDesignations(this.name)
+    let { allDesignationsStripped } = this
+    if (this.nameActionWords.length > 0) {
+      for (let word of this.nameActionWords) {
+        nameTest = replaceWord(nameTest, word)
+        allDesignationsStripped = replaceWord(allDesignationsStripped, word)
+      }
+    }
+    return (allDesignationsStripped === nameTest)
+  }
   get changesInBaseName () {
```

Comment:
This getter was previously living in grey-boxes.vue and had a watcher to update the store when its value changed because both this component (parent comp) and grey-boxes.vue (child component) relied on it. Obviously makes more sense to have it exist in parent and pass to child as a Prop.